packages/panzer/adapters-stk/test/assembly_engine/assembly_engine.cpp

```diff
@@ -593,12 +593,12 @@ namespace panzer {
     const Teuchos::RCP<panzer::ConnManager>
       conn_manager = Teuchos::rcp(new panzer_stk::STKConnManager(mesh));
 
-    panzer::DOFManagerFactory<int,panzer::Ordinal64> globalIndexerFactory;
-    RCP<panzer::UniqueGlobalIndexer<int,panzer::Ordinal64> > dofManager
-         = globalIndexerFactory.buildUniqueGlobalIndexer(Teuchos::opaqueWrapper(MPI_COMM_WORLD),physicsBlocks,conn_manager);
+    panzer::DOFManagerFactory globalIndexerFactory;
+    RCP<panzer::GlobalIndexer> dofManager
+         = globalIndexerFactory.buildGlobalIndexer(Teuchos::opaqueWrapper(MPI_COMM_WORLD),physicsBlocks,conn_manager);
 
     Teuchos::RCP<panzer::LinearObjFactory<panzer::Traits> > linObjFactory
-          = Teuchos::rcp(new panzer::TpetraLinearObjFactory<panzer::Traits,double,int,panzer::Ordinal64>(comm,dofManager));
+          = Teuchos::rcp(new panzer::TpetraLinearObjFactory<panzer::Traits,double,int,panzer::GlobalOrdinal>(comm,dofManager));
```

Comment:
There are a few of these Tpetra calls that use 'int' instead of 'panzer::LocalOrdinal'. Should we move over to LocalOrdinal, or are we removing these template parameters from the Tpetra interface next?

Reply:
We will eventually remove local ordinal completely, replacing with int. For now, where there were template parameters for local ordinals, I replaced with panzer::LocalOrdinal. If it already used int, I just left those. There are some subtleties in the dof-manager return types. I wanted to preserve that info temporarily to help with debugging in the applications if needed. We had a bunch of tests that exercised the LO template parameter by setting it to short int. Once we get the apps changed over, I can rename those to int before we push this if you want.

Reply:
Best practice with Tpetra is to use Tpetra's typedefs, like `Tpetra::Map<>::local_ordinal_type`.

Reply:
> Best practice with Tpetra is to use Tpetra's typedefs, like `Tpetra::Map<>::local_ordinal_type`.

@mhoemmen - I was under the impression that local ordinal was going away and being hard coded to int once deprecated code is removed. Are you keeping the typedef around?

Reply:
We'll keep the typedef around to "future-proof" -- allow changes to the default type in the future or allow future configure-time type options.  I agree with @mhoemmen's suggestion.